scripts/error_codes.py

```diff
@@ -227,10 +227,10 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
             return False
 
     old_source_only_ids = {
-        "1584", "1823",
+        "1218", "1584", "1823",
         "1988", "2066", "2833", "3356",
         "3893", "3996", "4010", "4458", "4802",
-        "4902", "5272", "5622", "5798", "7128", "7400",
+        "4902", "5272", "5622", "5798", "5840", "7128", "7400",
```

Comment:
These 2 error IDs were SMT errors when the solver couldn't prove something, which surprisingly doesn't happen anymore with the new z3! It's a nice development. We should anyway add some more complex tests that cause that in a future PR.